### ISSUE TITLE
Screen reader stops anouncing description after encountering certain characters.

### ISSUE THREAD

**Issue by jedrzejruta** [highcharts/highcharts#19002]
#### Expected behaviour
Screen reader should read out `<` character as _less_ or _less than_ depending on context.

#### Actual behaviour
When `<` character is encountered, screen reader outputs general phrase _picture_ and ignores provided description.

#### Live demo with steps to reproduce
[https://jsfiddle.net/BlackLabel/yk7zrpda/](https://jsfiddle.net/BlackLabel/yk7zrpda/)

#### Product version
Highcharts v11

#### Affected devices
iOS 16.4.1 with VoiceOver enabled
Windows 11 22H2 with NVDA

**Comment by jedrzejruta** [comment]
@oysteinmoseng - FYI

**Comment by oysteinmoseng** [comment]
Minimal demo: https://jsfiddle.net/oysteinmoseng/n5uk8yha/ 
Happens because of stripping HTML tags from labels.

**Comment by oysteinmoseng** [comment]
Workaround: https://jsfiddle.net/oysteinmoseng/br3zdo51/

Set `aria-label` manually after render.
